execution-order/execution-order.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html id="stylus">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title i18n-text-append="optionsHeading">Stylus </title>
+  <link rel="stylesheet" href="/global.css">
```

Comment:
Our dom.js needs a fix now in https://github.com/openstyles/stylus/blob/master/js/dom.js#L488, e.g. change `^` to `$`:

```js
const {sheet} = $('link[href$="global.css"]');
```